framerail/src/routes/[x+2d]/logout/+server.ts

```diff
@@ -0,0 +1,24 @@
+import { authLogout } from "$lib/server/auth/logout"
+
+export async function DELETE(event) {
+  let sessionToken = event.cookies.get("wikijump_token")
+
+  try {
+    let res = await authLogout(sessionToken)
+
+    event.cookies.set("wikijump_token", "", {
+      path: "/",
+      maxAge: 0
```

Comment:
I think you can just do `event.cookies.delete()` instead

Reply:
I think I tried that, but for whatever reason it didn't actually delete the cookie, so I resorted to maxAge 0 instead.

Reply:
Weird. Maybe leave a comment noting that.